manifests/weave/base/validator.yaml

```diff
@@ -77,7 +77,6 @@ spec:
              cat /data/tendermint/config/priv_validator_work.json | jq ".LastSignState" > /data/tendermint/data/priv_validator_state.json
              cat /data/tendermint/config/priv_validator_key.json | jq ".pub_key" | jq ". as \$k | {pub_key: \$k, power: \"10\", name: \"validator\"}" > ${TMHOME}/tmp/pub_validator.json
              cat ${TMHOME}/config/genesis.json | jq ".validators |= .+ [$(cat ${TMHOME}/tmp/pub_validator.json)]" > ${TMHOME}/config/genesis.json
-             rm -rf ${TMHOME}/tmp
```

Comment:
Maybe the check in L68 could be fixed instead? I don't think the tmp folder is of any use. Just check `data/tendermint/config` exists.

Reply:
done, I have implemented a better and more straight-forward solution.